### DISCUSSION TITLE
Global search (all objects) and Overlays

### DISCUSSION THREAD

**Discussion by nomad-cyanide** [netbox-community/netbox#10812] [General]
Hi

We have just upgraded to 3.3.2 and documented all of our L2VPN's and EVPN's in the new overlay module (which is awesome btw). But searching for an L2VPN or EVPN in the global search field at the top of the main page, searching in all objects does not produce a result. Overlay/L2VPN is not possible to choose as specific object either. 
It's the same behavior on 3.3.6, which our test-server is running.

Is anybody else seeing the same behavior?

**Comment by dejantep** [comment]
As far as i can see you can search for L2VPN Name but not identifier which i would like to see be implemented. I run 3.5.7
